### PR TITLE
fix(loader-utils): Fix incorrect browser field for 'utils'

### PR DESCRIPTION
**Pull request by ibgreen**

### CODE REVIEW

**Review by belom88 — CHANGES_REQUESTED** [review]
I would create a 3.3-release branch and make changes related to the version tag there. Because the master will be used for the next beta

**Comment by ibgreen** [comment]
> I would create a 3.3-release branch and make changes related to the version tag there. Because the master will be used for the next beta

I was thinking to create the 3.3-release branch in a day or two when we have been landing the initial round of fixes, as these will need to be cherry-picked to master anyway. But if you have something big that needs to land on master, we can cut the release branch now.

**Comment by belom88** [comment]
> > I would create a 3.3-release branch and make changes related to the version tag there. Because the master will be used for the next beta
> 
> I was thinking to create the 3.3-release branch in a day or two when we have been landing the initial round of fixes, as these will need to be cherry-picked to master anyway. But if you have something big that needs to land on master, we can cut the release branch now.

Ok. Let's land it to the master for now

File: modules/worker-utils/src/lib/worker-api/get-worker-url.ts [hunk 1]
```diff
@@ -1,8 +1,10 @@
+// loaders.gl, MIT license
+
 import type {WorkerObject, WorkerOptions} from '../../types';
 import {assert} from '../env-utils/assert';
 import {VERSION as __VERSION__} from '../env-utils/version';
 
-const NPM_TAG = 'beta'; // Change to 'latest' on release-branch
+const NPM_TAG = 'latest'; // 'beta', or 'latest' on release-branch
```
[the remark this answers is not in the document]
Comment (ibgreen) [reply modules/worker-utils/src/lib/worker-api/get-worker-url.ts:7]: > I would do these changes in the release branch

OK, rebased on 3.3-release.

**Pull request merged**